packages/server/src/api/controllers/datasource.ts

```diff
@@ -153,6 +154,21 @@ export async function verify(
   }
 }
 
+export async function fetchTables(
```

Comment:
Now we will have:

```ts
/api/datasources/:id:tables -> datasourceController#fetchTables
```
and
```ts
/api/tables -> tableController#fetch
```

Personally I feel like these entities are so similar it's not worth splitting like this, it'll likely be confusing why both exist.


Preference would be to augment the existing tables endpoint instead, to make it smarter so that we can query by datasource.
```ts
/api/tables?datasourceId=<id>
```

 If the `schema` is a problem we can make it optional. or use another param `noSchema` to make it more obvious.

Reply:
It's actually confusing, but I think they are different and maybe we should rename some. AFAIK, `/tables` are the the tables that a specific datasource has in budibase, while this new `/datasource/tables` are the tables available on the actual source

Reply:
Yes these are very different endpoints - the fetch tables really is just getting the table names - I wasn't really sure how to demonstrate that.

Perhaps a better name for this would be `information` - which retrieves initial information about the datasource+ connection, like what is available.

Reply:
This has been updated to an `info` endpoint - to reflect its relatively unrelated to all other table endpoints.